source/renderer/app/containers/staking/StakePoolsListPage.js

```diff
@@ -75,11 +76,11 @@ export default class StakePoolsListPage extends Component<Props> {
           selectedDelegationWalletId={selectedDelegationWalletId}
           stake={stake}
           onDelegate={this.handleDelegate}
-          isLoading={!isSynced || fetchingStakePoolsFailed}
-          isRanking={isRanking && stakePoolsRequest.isExecuting}
+          isLoading={isLoading}
+          isRanking={isRanking}
           getStakePoolById={getStakePoolById}
         />
-        {isRanking && stakePoolsRequest.isExecuting && (
+        {!isLoading && isRanking && stakePoolsRequest.isExecuting && (
```

Comment:
You can add `!isLoading` to the `isRanking` const and remove `stakePoolsRequest.isExecuting` from here, as it's already checked above.

Reply:
@daniloprates @yakovkaravelov resolved in: https://github.com/input-output-hk/daedalus/pull/2144/commits/e682113e7e5406b6ff60a064e6719facbb677699